### ISSUE TITLE
Binary Perl package is not depot-portable

### ISSUE THREAD

**Issue by mjtko** [alces-software/packager-base#77]
The `perl` binary(/ies) and `a2p` have hard-coded depot paths in the default search path -- when imported to a different depot the distributed perl libraries cannot be located.  Couple of options to experiment with to fix this:
1. explicitly add what are usually the "default" search paths to `PERL5LIB`.
2. look into implementing some kind of binary patching within the gridware tool.

**Comment by mjtko** [comment]
Default search path can be added using something like this:

```
append-path PERL5LIB ${appdir}/lib/site_perl/${version}/x86_64-linux
append-path PERL5LIB ${appdir}/lib/site_perl/${version}
append-path PERL5LIB ${appdir}/lib/${version}/x86_64-linux
append-path PERL5LIB ${appdir}/lib/${version}
```

**Comment by mjtko** [comment]
This was fixed in 35334c0dac66eadfba7166783e6dbf36bde8c223.
